kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeManagePanel.js

```diff
@@ -34,9 +34,9 @@ define([
 ) {
     'use strict';
     return new KBWidget({
-        name: "kbaseNarrativeManagePanel",
+        name: 'kbaseNarrativeManagePanel',
```

Comment:
This is the only diff that concerns me; there's a lot in here.  But if it seems okay let's just merge it.